docs/development/setup-advanced.md

```diff
@@ -278,5 +278,58 @@ likely only a few lines of changes to `tools/lib/provision.py` and
 `scripts/lib/setup-apt-repo` if you'd like to do it yourself and
 submit a pull request, or you can ask for help in
 [#development help](https://chat.zulip.org/#narrow/stream/49-development-help)
-on chat.zulip.org, and a core team member can help guide you through
-adding support for the platform.
+in [the Zulip development community](https://zulip.com/developer-community/),
+and a core team member can help guide you through adding support for the platform.
+
```

Comment:
When you rebased and resolved conflicts, you incorrectly included some changes that should not be a part of this commit. Everything starting with `## Installing on Cloud9` should be removed here.

Reply:
> When you rebased and resolved conflicts, you incorrectly included some changes that should not be a part of this commit. Everything starting with `## Installing on Cloud9` should be removed here.

I thought incoming change was the latest and I need to update my codebase, so I accepted the incoming change :')